src/@types/index.ts

```diff
@@ -358,19 +358,21 @@ export namespace RiotAPITypes {
     }
 
     export interface LeagueEntryDTO {
-      leagueId: string;
+      leagueId?: string;
```

Comment:
Since we have `Only included for the RANKED_TFT_TURBO` and `Not included for the RANKED_TFT_TURBO` we need to set some props as nullable.

https://developer.riotgames.com/apis#tft-league-v1/GET_getLeagueEntriesForSummoner

Reply:
I think you are updating the wrong `LeagueEntryDTO` as the one for `TFTLeague` is different to the one for normal `LeagueDTO`.

The `TFTLeague` types are found here: https://github.com/fightmegg/riot-api/blob/master/src/%40types/index.ts#L1344

Reply:
Thanks, fixed.